initramfs-init.in

```diff
@@ -364,6 +364,11 @@ if [ "${s390x_net%%,*}" = "qeth_l2" ]; then
 	echo 1 > /sys/bus/ccwgroup/drivers/qeth/"${_channel%%,*}"/online
 fi
 
+if [ -n "$ssh_install" ]; then
+	echo "nameserver $nameserver" > /etc/resolv.conf
```

Comment:
This is buggy. You probably mean `$KOPT_nameserver` here. But we may need to set the name server even if `ssh_install` (or `ssh_key`/`ssh_pass`) is unset, so it cannot be within the conditional. A better place may be to add it to the `ip=....` variable.